apps/full-site-editing/full-site-editing-plugin/global-styles/class-global-styles.php

```diff
@@ -78,7 +78,7 @@ class Global_Styles {
 	 */
 	private $plugin_name = 'jetpack-global-styles';
 
-	const VERSION = '1909241817';
+	const VERSION = '2003121439';
```

Comment:
@nosolosw could we use the plugin version constant here so this doesn't have to be updated for each change?

Reply:
This is only used for when the font list or CSS custom properties are updated so it's intentionally disconnected from plugin releases (no need to clean the cache if no changes to fonts).

Reply:
Would it work to have cache buster as some kind of hash from `AVAILABLE_FONTS` const, maybe combined with CSS file timestamp?